yt_dlp/extractor/ceskatelevize.py

```diff
@@ -119,12 +119,10 @@ def _real_extract(self, url):
                         type_ = 'bonus'
             if not idec:
                 raise ExtractorError('Failed to find IDEC id')
-            iframe_hash = self._download_webpage(
-                'https://www.ceskatelevize.cz/v-api/iframe-hash/',
-                playlist_id, note='Getting IFRAME hash')
-            query = {'hash': iframe_hash, 'origin': 'iVysilani', 'autoStart': 'true', type_: idec, }
+            sidp = playlist_id.rsplit('-')[0]
+            query = {'origin': 'iVysilani', 'autoStart': 'true', 'sidp': sidp, type_: idec, }
```

Comment:
```suggestion
            query = {'origin': 'iVysilani', 'autoStart': 'true', 'sidp': sidp, type_: idec}
```

Reply:
OK. I left the comma there to preserve existing style but I guess there is no reason for the trailing comma if the likst is in one line.